src/tests/madara_comparison.rs

```diff
@@ -12,12 +12,12 @@ fn trie_height_251() {
     let tempdir = tempfile::tempdir().unwrap();
     let db = create_rocks_db(tempdir.path()).unwrap();
```

Comment:
This test will only be runable with `std`.
Try run cargo test --no-default-features and see what bream

Reply:
It requires rocksdb which is not no std. It also requires pathfinder which i don't think is no std so no way to have those no std ?

Reply:
yeah that's my point. This test is not runnable in no_std. So it should be behind a feature flag.

Reply:
Right now if you do `run cargo test --no-default-features ` the compilation should fail

Reply:
done